src/math/Box3.js

```diff
@@ -215,69 +268,15 @@ Object.assign( Box3.prototype, {
 
 	},
 
-	expandByObject: function () {
-
-		// Computes the world-axis-aligned bounding box of an object (including its children),
-		// accounting for both the object's, and children's, world transforms
-
-		var scope, i, l;
-
-		var v1 = new Vector3();
-
-		function traverse( node ) {
-
-			var geometry = node.geometry;
-
-			if ( geometry !== undefined ) {
-
-				if ( geometry.isGeometry ) {
-
-					var vertices = geometry.vertices;
-
-					for ( i = 0, l = vertices.length; i < l; i ++ ) {
-
-						v1.copy( vertices[ i ] );
-						v1.applyMatrix4( node.matrixWorld );
-
-						scope.expandByPoint( v1 );
+	expandByObject: function ( object ) {
 
-					}
+		object.updateMatrixWorld( true );
 
-				} else if ( geometry.isBufferGeometry ) {
+		object.traverse( traverse.bind( this ) );
```

Comment:
I'm analyzing the closures in `Box3` now. Are you aware that `bind()` returns a new function on each invocation? It's the same like defining `traverse()` directly in `expandByObject()`.

I'm afraid we need some refactoring here...